### ISSUE TITLE
[BUG] - ChatAliases / Can't enter settings

### ISSUE THREAD

**Issue by JustCryen** [mwittrien/BetterDiscordAddons#2090]
**Describe the bug**
Since quite a long time I was unable to enter settings for this plugin.
I thought it was already known but as I just looked, no one posted a bug report about this yet.

**To Reproduce**
Try to enter ChatAliases Settings from the Plugins > cogwheel
![image](https://user-images.githubusercontent.com/41633661/204921306-f12bf095-9750-4a91-b5c6-32ccb4e9b733.png)
Console says:
![image](https://user-images.githubusercontent.com/41633661/204921602-19f207f0-607d-4a00-bf98-d1d4032c7ed1.png)

**Expected behavior**
I should be able to enter settings and edit the aliases just like before.

**Discord Build:**
PTB 161447 (b8ed591)

**Additional context**
Besides that the plugin works. 
I just can't edit or add anything since I cannot access settings.

**Comment by mwittrien** [comment]
It hasnt been fixed yet

**Comment by JustCryen** [comment]
Right, I think it was still worth adding the bug report to keep track of it.
Also, editing `ChatAliases.config.json` directly is a good alternative for now.

I hope you'll have some time to fix this!
Maybe I'll try to poke at it as well ;)

**Comment by JustCryen** [comment]
Oh woah, that was fast, Thanks!
I've looked at the fix and it kinda explains how you managed to do it so fast xP
I wonder how it even worked before…
Anyway, thanks again!
